app/views/errors/application_not_found.html.erb

```diff
@@ -0,0 +1,11 @@
+<% title t('.page_title') %>
+
```

Comment:
I'm wondering if we could just have a generic view for - "Application not found" "not found" and "unhandled" as the ERB partials appear to be identical. Seems like we have two types of view error partial:

- concise
- verbose (i.e. has the `.more_text`) `<p>` tag

🤔  

I guess the benefit of having them separate is that the i18n works, but we could compose them down into either concise or verbose...

Just a thought its not much code to duplicate really.

Reply:
Yes, having separate views helps a lot with the localisation. Now they look similar, but it may be they require further customisation down the line, so didn't want to anticipate the future, for now is just easier having 1 view per error.

But definitely if we end up creating many more error pages, all almost identical, we can extract some bits DRY some bits 👍

Reply:
Makes sense! 👍